api/controller/readcsv_controller.go

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import (
+	model "api/models"
+	"encoding/csv"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/unrolled/render"
+)
+
+func ReadCSV(w http.ResponseWriter, r *http.Request) {
+
+	var resp = render.New()
+	var Contestants []model.Contestant
+
+	csvf, err := os.Open("../api/lmd.csv")
+	if err != nil {
+		fmt.Println(err)
+		resp.JSON(w, http.StatusBadRequest, map[string]string{"error": err.Error()})
```

Comment:
I don't think a BadRequest status code would be appropriate for this error, it wasn't the users fault that the file couldn't be opened. Maybe a 500?

Reply:
Also you wouldn't like to return this type of errors to the user, it's ok to log them, but returning them could give away details on how your app works. You could return something like `Something happened while processing your request, try again`